state-chain/pallets/cf-broadcast/src/tests.rs

```diff
@@ -965,22 +1006,69 @@ fn broadcast_can_be_aborted_due_to_time_out() {
 		});
 }
 
+#[test]
+fn broadcast_timeout_works_when_external_chain_advances_multiple_blocks() {
+	new_test_ext()
+		.execute_with(|| {
+			let mut broadcast_ids = Vec::new();
+			let (broadcast_id, _) = start_mock_broadcast();
+			broadcast_ids.push(broadcast_id);
+			ready_to_abort_broadcast(broadcast_id);
+
+			let expiry = BlockHeightProvider::<MockEthereum>::get_block_height()
+				.saturating_add(crate::BroadcastTimeout::<Test, Instance1>::get());
+
+			(broadcast_ids, expiry)
```

Comment:
nit - no need to mut initialise and push here
```suggestion
			(vec![broadcast_id], expiry)
```